conf/resinhup.conf

```diff
@@ -54,6 +55,7 @@ to_keep_files:
 ; persistent over updates.
 ; Only relative paths to boot partition.
 ignore_files:
```

Comment:
`ignore_files` is a list of files which we want to persist across updates. Basically we won't copy these files from the update package. `boot_whitelist` is a list used when checking the fingerprints. Even though they include the same list they have different usage and one is an absolute path one is relative. We can definitely merge them in one as I can't see a case where these would be different.